packages/packages.go

```diff
@@ -50,8 +51,15 @@ var (
 
 	// DebugLogger is the debug logger to use.
 	DebugLogger = log.New(ioutil.Discard, "", 0)
+
+	// Runner is the CommandRunner used for running exec commands.
+	Runner util.CommandRunner
```

Comment:
Does this have to be public? Can the mocks not just be generated privately for this library?

Reply:
in order to inject the mockrunner, this has to be public since the injection happens in the test package

Reply:
ignore the last comment. that is not true. however, it had to be public otherwise my tests failed with panic, and changing the scope resolved it.

Reply:
sorry about the confusions from previous commments. i think i merged 2 issues that i faced yesterday

it does not have to public. changed it